renderer/store/db.js

```diff
@@ -138,6 +138,9 @@ export const getDb = name => {
     // Save the migrated config overrides.
     await tx.settings.put({ key: 'config', value: newConfig })
   })
+  db.version(5).stores({
```

Comment:
No need for new db table, right @korhaliv ?

Reply:
Right, bad merge result